src/integrations/prefect-gcp/prefect_gcp/workers/cloud_run_v2.py

```diff
@@ -226,7 +226,10 @@ def _configure_cloudsql_volumes(self):
         if "volumes" not in template:
             template["volumes"] = []
         template["volumes"].append(
-            {"name": "cloudsql", "cloudSqlInstance": self.cloudsql_instances}
+            {
+                "name": "cloudsql",
+                "cloudSqlInstance": {"instances": self.cloudsql_instances},
```

Comment:
can you link a reference that motivates this change?

Reply:
```
"GoogleCloudRunV2CloudSqlInstance": {
      "id": "GoogleCloudRunV2CloudSqlInstance",
      "description": "Represents a set of Cloud SQL instances. Each one will be available under /cloudsql/[instance]. Visit https://cloud.google.com/sql/docs/mysql/connect-run for more information on how to connect Cloud SQL and Cloud Run.",
      "type": "object",
      "properties": {
        "instances": {
          "description": "The Cloud SQL instance connection names, as can be found in https://console.cloud.google.com/sql/instances. Visit https://cloud.google.com/sql/docs/mysql/connect-run for more information on how to connect Cloud SQL and Cloud Run. Format: {project}:{location}:{instance}",
          "type": "array",
          "items": {
            "type": "string"
          }
```

[gcp discovery api cloud run v2 docs](https://run.googleapis.com/$discovery/rest?version=v2)

The `instances` key is required. The current  `prefect-gcp` implementation of the sqlcloud connection does  not work.